utilities/mtv_migration.py

```
@@ -542,3 +542,74 @@ def verify_vm_disk_count(destination_provider, plan, target_namespace):
+
+    Args:
+        plan_list: List of Plan resources to monitor.
+        timeout: Timeout in seconds to wait for simultaneous execution.
+
+    Returns:
+        None
+
+    Raises:
+        AssertionError: If plans do not execute simultaneously or if any plan completes early.
+    """
+    LOGGER.info(f"Validating {len(plan_list)} migrations enter executing state simultaneously")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)